gammapy/analysis/config/config.yaml

```diff
@@ -36,7 +36,7 @@ datasets:
         method: reflected
         exclusion:    # fits file for exclusion mask
     # mandatory for 1d
```

Comment:
I suggest to remove all these "mandatory for 1d" and "mandatory for 3d" comments.
Two reasons:
- It's not clear what they mean. Currently we have defaults, so it's not mandatory to set them in user config files, saying those are "mandatory" isn't true. I guess you mean "used for 1d" and "used for 3d"?
- We will very likely completely restructure this 1D / 3D config options in the next days, and very quickly any comments saying what is and isn't used for 1D / 3D will turn into lies. We might add back these comments once the dust has settled, but more likely we'll try to come up with a schema that is clear by itself, so that we don't have to rely on comments to communicate this information to the users.

Reply:
Done.